### PR TITLE
rebar2 compatibility

### PR DESCRIPTION
**Pull request by brigadier**
`rebar.config.script` file to make it possible to compile the app with rebar2.

### CODE REVIEW

**Comment by id** [comment]
LGTM. @zmstone ?

File: rebar.config.script [hunk 1]
```diff
@@ -0,0 +1,12 @@
+IsRebar3 = erlang:function_exported(rebar3, main, 1),
+Rebar2Deps = [
```
Comment (zmstone) [inline rebar.config.script]: could you take the version tags from rebar.config instead of repeating here ?

**Comment by brigadier** [comment]
> could you take the version tags from rebar.config instead of repeating here ?

done

**Comment by zmstone** [comment]
good stuff!
thanks!

**Pull request merged**